README.md

```diff
@@ -64,12 +64,14 @@ account in `security.json` is user `solr` using the password `SolrRocks`.
 
 To access the Solr container from DDEV's web container, use  `http://solr:8983`.
 
+More at [Drupal at your fingertips: Solr and DDEV](https://www.drupalatyourfingertips.com/development#solr-and-ddev).
```

Comment:
Does this belong in the Drupal section instead of here? (Note this add-on is intended for any solr consumer)

Reply:
No, I think this is appropriate as I show solr screens and such without much Drupal-goodness

Reply:
Removed